src/Steam.package/AppDescriptorSystem.class/instance/allTableNames.st

```diff
@@ -0,0 +1,9 @@
+accessing
+allTableNames
+	| result |
+	result := OrderedCollection new.
+	STObject allSubclasses do: [ :e |
+		|title|
+		title := e asString asUppercase.
+		(title includesSubstring: 'SEARCH') ifFalse: [ result add: title ].].
+	^ result asArray
```

Comment:
At some point this AppDescriptorSystem should be removed, isn't it?

Reply:
Yes, for now it's using this class. 
But i'm working on STDescriptorSystem to remplace the first one